syslog/syslog.c

```diff
@@ -355,13 +360,14 @@ syslog_add_entry(syslog_entry_t *entry)
       pse = pse->next;
     pse->next = entry;	// append msg to syslog queue
   }
-//   DBG("%p %lu %d\n", entry, entry->msgid, system_get_free_heap_size());
+  DBG("%p %lu %d %s\n", entry, entry->msgid, system_get_free_heap_size(), entry->datagram);
```

Comment:
This should be commented out, no? What it prints is not self-explanatory at all.